src/bioregistry/data/bioregistry.json

```diff
@@ -14623,6 +14623,38 @@
       "uri_format": "https://www.cellrepo.com/repositories/$1"
     }
   },
+  "cellxgene": {
```

Comment:
I think this should be renamed to `cellxgene.collection`, since it's not obvious from the name what kind of entity type this refers to, and there might be other cellxgene resources added later